src/components/SelectNodeId.svelte

```diff
@@ -72,6 +72,9 @@
       { label: "Manual", value: "manual" }
     ]
   };
+  $: {
+    nodeSelectionField.disabled = validating;
```

Comment:
Please remove reactive statement from here as it's not needed (will add alternative code below)